### ISSUE TITLE
Default embedded keys are slowing dump speed

### ISSUE THREAD

**Issue by unknown10777** [GameTec-live/ChameleonUltraGUI#384]
**Is your feature request related to a problem? Please describe.**
tl;dr: sector 0,1 are encrypted, need recover keys, but it tried 52 keys even I select "Empty" dict.
it costs 5 sec each sector so I finally can recover keys.

In my surrounding, the cards only use few sectors store encrypted data. So when I try dump card, it read and check keys from "Empty" dict, but cost much time try read unknown sectors. I can't recover keys until it finally done.
![image](https://github.com/GameTec-live/ChameleonUltraGUI/assets/83267603/30f4ea72-bb39-456c-96b6-976199cd37e3)

**Describe the solution you'd like**
My opinion is try default `ffffffffffff` only to speed up reading, then if user want, try extended dict, or recover the keys directly.
But not showing "Empty (0 keys)" and still using embedded 52 keys. This really cost time (3s usb, 6s ble, per block).

**Comment by GameTec-live** [comment]
"Empty" is in this case a small dictionary with only the most common keys (ik, it should be renamed). If you already know the key, then you could just create your own dictionary with only that key...

**Comment by GameTec-live** [comment]
Altough adding a skip button to directly skip to recovery might be smthng... Just stopping in the middle of the dict check seems impractical though and would probably result in some very wierd cardsaves...

**Comment by GameTec-live** [comment]
someone else alao wanted a button to skip the dict checks and go to recovery, so ig, why not 🤷

**Comment by Foxushka** [comment]
Its fast, no

**Comment by GameTec-live** [comment]
checking an entire dict just for it to not find them can over BLE ve very slow
